src/components/TableModule/TableHeaderCell.tsx

```diff
@@ -130,7 +141,9 @@ export const TableHeaderCell: React.FC<TableHeaderCellProps> = ({
         ((!header.align && headingsCount > 1 && index === headingsCount - 1) ||
           header?.align === 'right') &&
           classes.rootAlignRight,
-        header.className
+        header.className,
+        isSticky && classes.isSticky,
+        isSticky && 'sticky-cell-hook'
```

Comment:
what is `sticky-cell-hook` used for? Just tests? It might be better to use a `data-*` attribute like `data-sticky` for example.

Reply:
It's also used to apply the isStickyLast class to the last sticky cell in each row to form the last column styling with the border (see the useEffect on line 323 of TableModule.tsx). I did this since we can't directly select the MUI isSticky class since its ID changes between builds.

Now I'm realizing I got around this with regex in the tests - which method do you think would be better?

Reply:
personally I'd go with what you did for isStickyLast, just for consistency, or else I'd use a testid/data attribute, but honestly it's not a major deal either way.